install.sh

```diff
@@ -19,6 +19,21 @@ fi
 
 XPUI_PATH="${INSTALL_PATH}/Contents/Resources/Apps"
 
+# Script flags
+CACHE_FLAG=''
```

Comment:
If `CACHE_FLAG` is a boolean, we can default it to `false` here instead of empty string.

Reply:
Sounds good. It does seem more proper that way. Updated to `false` rather than leaving empty.